src/composables/useSignRelayerApproval.ts

```diff
@@ -23,6 +24,9 @@ export default function useSignRelayerApproval(relayerType: RelayerType) {
   const { account, getSigner } = useWeb3();
   const { networkId } = useNetwork();
   const { t } = useI18n();
+  const { action: gnosisApprovalAction } = useRelayerApproval(
+    RelayerType.GNOSIS
```

Comment:
What if we change this to `Relayer`, i.e. the type of approval which is requested, not just Gnosis (Cowswap). This will include batch relayer.